vault/logical_system_paths.go

```diff
@@ -345,6 +345,47 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 }
 
 func (b *SystemBackend) rekeyPaths() []*framework.Path {
+	respFields := map[string]*framework.FieldSchema{
+		"nounce": {
+			Type:     framework.TypeString,
+			Required: true,
+		},
+		"started": {
+			Type:     framework.TypeString,
+			Required: true,
+		},
+		"t": {
+			Type:     framework.TypeInt,
+			Required: true,
+		},
+		"n": {
+			Type:     framework.TypeInt,
+			Required: true,
+		},
+		"progress": {
+			Type:     framework.TypeInt,
+			Required: true,
+		},
+		"required": {
+			Type:     framework.TypeInt,
+			Required: true,
+		},
+		"verification_required": {
+			Type:     framework.TypeBool,
+			Required: true,
+		},
+		"verification_nonce": {
+			Type:     framework.TypeString,
+			Required: true,
+		},
+		"backup": {
+			Type: framework.TypeBool,
+		},
+		"pgp_fingerprints": {
+			Type: framework.TypeCommaStringSlice,
+		},
```

Comment:
Are these fields optional? If so, might be good to mark them as `Required: false` for consistency.

Reply:
Oh I thought the Hashi best practice was to use the golang zero value? should I at that everywhere?

Reply:
🤷  I guess it's OK either way. To me it seemed more clear with Require: false for emphasis but leaving it out is fine too.